client/web/src/enterprise/code-monitoring/CodeMonitoringNode.scss

```
@@ -0,0 +1,23 @@
+.code-monitoring-node {
+    cursor: pointer;
+    // stylelint-disable-next-line declaration-property-unit-whitelist
+    margin: -1px -1px calc(0.5rem - 1px) -1px;
+
+    &:hover {
+        border: 2px solid var(--link-color);
```

```suggestion
        border: 2px solid #329af0;
```

There ought be a colour variable for `$color-border-active`, but I'm not seeing it defined in the global styles. Either way, the hover state is a lighter blue than the `--link-color`. This is what's used for the cards in the code monitor form.